mpas_analysis/ocean/time_series_ohc.py

```diff
@@ -378,6 +383,19 @@ def run(self):  # {{{
                                       colorbarLabel, title, xLabel, yLabel,
                                       figureName, linewidths=1, xArrayIsTime=True,
                                       N=movingAveragePointsHovmoller, calendar=calendar)
+
+                caption = 'Trend of {} Temperature vs depth'.format(region)
+                write_image_xml(
+                    config=config,
+                    filePrefix=self.filePrefixes['TZ', region],
+                    componentName='Ocean',
+                    componentSubdirectory='ocean',
+                    galleryGroup='Trends vs Depth',
+                    groupLink='trendsvsdepth',
+                    thumbnailDescription=u'{} ΔT'.format(region),
```

Comment:
@gstreletz, could you take out the Deltas here and in the other thumbnail descriptions you added here?  These aren't anomalies.  Thanks!